slash/css/src/Form/core/FormCore.scss

```
@@ -1,5 +1,7 @@
 @use "../../common/common" as common;
 
+$legend-flex-width: calc(100% / 6);
```

il est préférable d'utiliser les tokens css

---

Je ne suis pas sur de comment ça fonctionne. Il n'y a pas de token pour la largeur relative, je vais regarder pour en ajouter un.

---

tu peux le créer en faisant : 
```suggestion
--legend-flex-width: calc(100% / 6);
```

---

C'est fait.